A-concurrency-pipeline.md

```diff
@@ -52,7 +52,7 @@ Semoga cukup jelas ya. Gpp kalau bingung, nanti kita sambil praktek juga jadi bi
 
 Ok, penjabaran teori sepanjang sungai `nil` tidak akan banyak membawa penjelasan yang real kalau tidak diiringi dengan praktek. So, mari kita mulai praktek.
 
-Penulis sarankan untuk benar-benar memahami setiap bagian praktek ini, karena topik ini merupakan pembahasan yang cukup berat untuk pemulai, tapi masih dalam klasifikasi fundamental kalau di Go programming. Bingung tidak apa, nanti bisa di-ulang-ulang, yang penting tidak sekadar *copy-paste*.
+Penulis sarankan untuk benar-benar memahami setiap bagian praktek ini, karena topik ini merupakan pembahasan yang cukup berat untuk pemula, tapi masih dalam klasifikasi fundamental kalau di Go programming. Bingung tidak apa, nanti bisa di-ulang-ulang, yang penting tidak sekedar *copy-paste*.
```

Comment:
The word `sekadar` is actually a valid Indonesian word :) Please revert this one. https://kbbi.web.id/sekedar%20atau%20sekadar

Reply:
aah i'm sorry for that, i will revert asap